src/plugins/walletConnectToDapps/WalletConnectBridgeProvider.tsx

```diff
@@ -215,8 +215,10 @@ export const WalletConnectBridgeProvider: FC<PropsWithChildren> = ({ children })
         accountNumber,
         to: tx.to,
         data: tx.data,
-        value: tx.value ?? '0',
-        gasLimit: approveData.gasLimit ?? tx.gas ?? '9000', // https://docs.walletconnect.com/1.0/json-rpc-api-methods/ethereum#eth_sendtransaction
+        value: tx.value ?? convertNumberToHex(0),
+        gasLimit:
+          (approveData.gasLimit ? convertNumberToHex(approveData.gasLimit) : tx.gas) ??
+          convertNumberToHex(9000), // https://docs.walletconnect.com/1.0/json-rpc-api-methods/ethereum#eth_sendtransaction
```

Comment:
should these all be `90000` rather than `9000`?

`gas: QUANTITY - (optional, default: 90000) Integer of the gas provided for the transaction execution. It will return unused gas.`

Reply:
Resolved in https://github.com/shapeshift/web/pull/3173/commits/d0daf6ca3065c01c15dc832a849a8decb707f0c1